ide/static/ide/js/ib/utils.js

```diff
@@ -31,11 +31,26 @@
         };
     };
 
-    IB.ColourWhite = new IB.Colour('GColorWhite', 'white', gettext('White'));
-    IB.ColourBlack = new IB.Colour('GColorBlack', 'black', gettext('Black'));
-    IB.ColourClear = new IB.Colour('GColorClear', 'rgba(0, 0, 0, 0)', gettext('Transparent'));
+    IB.ColourModes = {
+        Colour: 0,
+        Monochrome: 1
+    };
+    IB.ColourMode = null;
```

Comment:
I'm not sure that this should use the same naming convention is everything else, which is a constant. Does anything else do that?